fetch-todolist/src/component/AddToDoList.js

```diff
@@ -1,51 +1,67 @@
 import { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
 
-const AddToList = ({ handleAdd, openDiv }) => {
+const AddToList = ({ openDiv }) => {
   const [task, setTask] = useState("");
   const [ddate, setDdate] = useState("");
-
+  const navigate = useNavigate();
   const handleTask = (e) => {
     setTask(e.target.value);
   };
   const handleDdate = (e) => {
     setDdate(e.target.value);
   };
+  const activity = {
+    id: "",
+    task: task,
+    dueDtae: ddate,
+    status: false,
+  };
+  const handleAdd = () => {
+    fetch("http://localhost:8080/todo", {
```

Comment:
move api calls to separate files

Reply:
I have created a separate folder name services, in that I've made two files for api.